pkg/asset/installconfig/openstack/validation/machinepool_test.go

```diff
@@ -13,65 +13,72 @@ import (
 const (
 	validZone = "valid-zone"
 
-	validCtrlPlaneFlavor = "valid-control-plane-flavor"
-	validComputeFlavor   = "valid-compute-flavor"
+	validCtrlPlaneFlavorDisk      = "valid-control-plane-flavor-disk"
+	validCtrlPlaneFlavorEphemeral = "valid-control-plane-flavor-ephemeral"
+	validComputeFlavor            = "valid-compute-flavor"
 
 	notExistFlavor = "non-existant-flavor"
 
-	invalidComputeFlavor   = "invalid-compute-flavor"
-	invalidCtrlPlaneFlavor = "invalid-control-plane-flavor"
+	invalidComputeFlavor            = "invalid-compute-flavor"
+	invalidCtrlPlaneFlavorDisk      = "invalid-control-plane-flavor-disk"
+	invalidCtrlPlaneFlavorEphemeral = "invalid-control-plane-flavor-ephemeral"
 
 	baremetalFlavor = "baremetal-flavor"
 )
 
 func validMachinePool() *openstack.MachinePool {
 	return &openstack.MachinePool{
-		FlavorName: validCtrlPlaneFlavor,
+		FlavorName: validCtrlPlaneFlavorDisk,
 		Zones:      []string{""},
 	}
 }
 
 func validMpoolCloudInfo() *CloudInfo {
 	return &CloudInfo{
 		Flavors: map[string]Flavor{
-			validCtrlPlaneFlavor: {
+			validCtrlPlaneFlavorDisk: {
 				Flavor: &flavors.Flavor{
-					Name:  validCtrlPlaneFlavor,
-					RAM:   16,
-					Disk:  25,
-					VCPUs: 4,
+					Name:      validCtrlPlaneFlavorDisk,
+					RAM:       16,
+					Disk:      25,
+					VCPUs:     4,
+					Ephemeral: 0,
 				},
 			},
 			validComputeFlavor: {
 				Flavor: &flavors.Flavor{
-					Name:  validComputeFlavor,
-					RAM:   8,
-					Disk:  25,
-					VCPUs: 2,
+					Name:      validComputeFlavor,
+					RAM:       8,
+					Disk:      25,
+					VCPUs:     2,
+					Ephemeral: 0,
 				},
 			},
-			invalidCtrlPlaneFlavor: {
+			invalidCtrlPlaneFlavorDisk: {
 				Flavor: &flavors.Flavor{
-					Name:  invalidCtrlPlaneFlavor,
-					RAM:   8, // too low
-					Disk:  25,
-					VCPUs: 2, // too low
+					Name:      invalidCtrlPlaneFlavorDisk,
+					RAM:       8, // too low
+					Disk:      25,
+					VCPUs:     2, // too low
+					Ephemeral: 0,
 				},
 			},
 			invalidComputeFlavor: {
 				Flavor: &flavors.Flavor{
-					Name:  invalidComputeFlavor,
-					RAM:   8,
-					Disk:  10, // too low
-					VCPUs: 2,
+					Name:      invalidComputeFlavor,
+					RAM:       8,
+					Disk:      10, // too low
+					VCPUs:     2,
+					Ephemeral: 0,
 				},
 			},
 			baremetalFlavor: {
 				Flavor: &flavors.Flavor{
-					Name:  baremetalFlavor,
-					RAM:   8,  // too low
-					Disk:  10, // too low
-					VCPUs: 2,  // too low
+					Name:      baremetalFlavor,
+					RAM:       8,  // too low
+					Disk:      10, // too low
+					VCPUs:     2,  // too low
+					Ephemeral: 0,
```

Comment:
you don't technically need this.  Ephemeral is an int defined in the flavor.Flavor struct,  it will be initialized to 0 if you don't provide it, when isntantiating  a flavors.Flavor struct.

Reply:
ok, I'll remove them.